libglusterfs/src/syncop-utils.c

```diff
@@ -294,7 +294,7 @@ _run_dir_scan_task(call_frame_t *frame, xlator_t *subvol, loc_t *parent,
     int ret = 0;
     struct syncop_dir_scan_data *scan_data = NULL;
 
-    scan_data = GF_CALLOC(1, sizeof(struct syncop_dir_scan_data),
```

Comment:
Looks OK, though I wish lines below 304-314 were setting variables in the order they are in the struct.

Reply:
Updated